pkg/conversion/unversioned_test.go

```diff
@@ -0,0 +1,90 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package conversion_test
+
+import (
+	"encoding/json"
+	"reflect"
+	"testing"
+
+	"k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/api/testapi"
```

Comment:
On the package arrangement.

pkg/conversion was sort of intended to be a layer independent from the rest of the code, such that we could toss it in its own repository. Importing the api packages like this makes that impossible.

Can you add a TODO here, saying that it's pretty hacky to import api like this, and that ideally we should recreate the necessary package structure in e.g. `pkg/conversion/test/...`.

Alternatively, you could recreate the necessary package structure in e.g. `pkg/conversion/test/...` but that's a lot of work and we can probably defer it.